packages/v4/src/content/design-guidelines/content/numerics.md

```diff
@@ -8,7 +8,7 @@ This page provides best practices for expressing values including:
 - Date and time
 - Numbers and currency
 
-See our [Units and symbols](/design-guidelines/content/units-and-symbols) page to learn how to use numeric units and symbols in your UI.
+See our [Units and symbols](https://www.patternfly.org/v4/ux-writing/units-and-symbols) page to learn how to use numeric units and symbols in your UI.
```

Comment:
Don't use absolute links, use relative links in case we ever change the path prefix to be something other than `v4`. So change to `[Units and symbols](/ux-writing/units-and-symbols)`.

Reply:
Got it! I'll change these to relative ones 👍